src/routes/v2/partner.ts

```diff
@@ -352,7 +352,7 @@ router.post(
         .optional()
         .isEmail()
         .run(req);
-      await check("fee", "Fee is invalid").isNumeric().run(req);
+      await check("postFee", "postFee is invalid").isNumeric().run(req);
```

Comment:
postfee shouldn't be passed in from the pylon frontend, this will be assigned when creating an order via the API.
When starting a checkout we'll be getting the postfee value from the checkoutRequest.